tests/atest/rules/spacing/empty_lines_between_sections/bug861.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+Library     String
+
+# Comment
```

Comment:
bug861.robot:4:1 [W] 1003 Invalid number of empty lines between sections (1/2)

Shouldn't it be 0/2? If at all, it's problematic because some people want to have comments right above header


```
*** Settings ***


*** Keywords ***  # fine
```

```
*** Settings ***


# comment - should be fine?
*** Keywords ***
```

```
*** Settings ***


# comment

*** Keywords ***  1/2 issue
```

Reply:
In a `tests/atest/rules/spacing/empty_lines_between_sections/golden.robot` file we have a case:
```
*** Variables ***
${MY_VAR}    1


# some comment
*** Keywords ***
```

and the issue is not reported there. So I don't know 😁

Reply:
Maybe we should just simplify it and count from 2 empty lines after any kind of text (including comments)?

Reply:
Yeah, I'm for simplification

Reply:
But please check history of git - I have strangr feeling we discussed something similar not so long ago and thats why it is implemented this way

Reply:
Ok, found it. We discussed it in #611. And I think that if we align current implementation of 1003 rule to what was decided in the related issue, it should work exactly as I mentioned above - it should not apply fancy rules for comments and just ignore them and count right after the last comment. All other empty lines between comments will be handled by the 1005 rule.

Reply:
Changed, take a look.

Reply:
Now it looks fine!

Do you want it to be part of current fix release (not yet released 3.2.1) or another feature release (3.3.0)? Please update release docs with this change depending on which version you want to use. If you don't want it to include it in 3.2.1 I will make release before you merge this PR

Reply:
Since this is a fix, we can push it with the 3.2.1. I will adjust the release notes.

Reply:
Done. I think we're ready to release.